torch/_inductor/subgraph_lowering.py

```diff
@@ -63,11 +68,6 @@ def call_function(
                 f"{target} not supported in subgraph, (missing lowering)"
             )
 
-        if torch.Tag.pointwise not in target.tags:
```

Comment:
This is a bit restrictive, since there are operators like `aten.full` or `aten.index` that aren't "pointwise" but we can still lower.